### PR TITLE
Remove unused var in presto-kudu

### PR DESCRIPTION
**Pull request by brandboat**
#4188
This PR is a preparation before enable UnusedVariable error prone check.

### CODE REVIEW

File: presto-kudu/src/main/java/io/prestosql/plugin/kudu/properties/KuduTableProperties.java [hunk 1]
```diff
@@ -584,8 +582,7 @@ private static Number toNumber(Object obj, Type type, String name)
         }
         else if (obj instanceof String) {
             String s = (String) obj;
-            BigDecimal d = new BigDecimal((String) obj);
-            return d;
+            return new BigDecimal(s);
```
Comment (Praveen2112) [inline presto-kudu/src/main/java/io/prestosql/plugin/kudu/properties/KuduTableProperties.java:585]: Can we inline `(String) obj` ?

**Review by Praveen2112 — APPROVED** [review]
Minor comments

File: presto-kudu/src/main/java/io/prestosql/plugin/kudu/properties/KuduTableProperties.java (hunk 1 above)
Comment (brandboat) [reply presto-kudu/src/main/java/io/prestosql/plugin/kudu/properties/KuduTableProperties.java:585]: Sure. I will fix it later.

Comment (brandboat) [reply presto-kudu/src/main/java/io/prestosql/plugin/kudu/properties/KuduTableProperties.java]: Done.

**Comment by Praveen2112** [comment]
Merged ! Thanks for working on this

**Pull request merged**